homeassistant/components/homeassistant_connect_zbt2/manifest.json

```diff
@@ -20,5 +20,6 @@
       "pid": "831A",
       "vid": "303A"
     }
-  ]
+  ],
+  "usb_abort_discovery_on_unplug": true
```

Comment:
Should this default to true ?  It seems like we would want this by default, even if its a breaking change.

Reply:
I threw this commit in as a proposal for a way to keep this behavior opt-in (I'm fine breaking custom components here, USB discovery is very rare to begin with). There are a few contrived situations where this would break existing functionality.